### PR TITLE
Update GettingStarted screen

### PR DESCRIPTION
**Pull request by bmunkholm**
Remove "Beta" from "Beta Forums".
Rename "Tasks" to "Todo"
Add React Native button to jump to getting started.

This PR has not be tested! Please validate it's working correctly before merge.

### CODE REVIEW

**Comment by bmunkholm** [comment]
@nirinchev Please review - would be good to get included in next release.

**Comment by bmunkholm** [comment]
or @kraenhansen

**Review by nirinchev — CHANGES_REQUESTED** [review]

File: src/ui/ServerAdministration/GettingStarted/index.tsx [hunk 1]
```diff
@@ -60,6 +60,10 @@ class GettingStartedContainer extends React.Component<
       electron.shell.openExternal(
         'https://docs.realm.io/platform/v/3.x/getting-started/android-quick-start',
       );
+    } else if (name === 'react-native') {
+      electron.shell.openExternal(
+        'https://docs.realm.io/platform/v/3.x/getting-started/react-native-quick-start',
```
[the remark this answers is not in the document]
Comment (bmunkholm) [reply src/ui/ServerAdministration/GettingStarted/index.tsx]: Not really sure. 
This is currently what works when you just goto the website: https://docs.realm.io/platform/getting-started-1/react-native-quick-start
And the others also look like this. But for some reason, they seem to redirect well enough now from the "v/3.x/", but react-native doesn't...
Seems like this should be solved at the doc level. We need to have url's that doesn't change when we include them in programs.

[the remark this answers is not in the document]
Comment (kraenhansen) [reply src/ui/ServerAdministration/GettingStarted/index.tsx]: Should we merge with this instead?

```suggestion
        'https://docs.realm.io/platform/getting-started-1/react-native-quick-start',
```

[the remark this answers is not in the document]
Comment (bmunkholm) [reply src/ui/ServerAdministration/GettingStarted/index.tsx]: Only if we can ensure this link lives "forever".
I'm not sure what the strategy is for those urls. Perhaps discuss this with @mgeerling in Slack #docs where we had some discussion about it.

[the remark this answers is not in the document]
Comment (mgeerling) [reply src/ui/ServerAdministration/GettingStarted/index.tsx]: @bmunkholm if you want to ensure that this lives forever, feel free to take a look at how redirects work for gitbook as I am not going to have time to do this. 

Those are the links right now.  There is going to be a URL change in the next few months as part of this docs restructure.  After this major restructure, it is unlikely that they will change again.

[the remark this answers is not in the document]
Comment (bmunkholm) [reply src/ui/ServerAdministration/GettingStarted/index.tsx]: @mgeerling I'll be happy to help with that. But in the context of the docs restructure project ensuring that these redirects happen needs to be part of that project. Again happy to help if I can.

[the remark this answers is not in the document]
Comment (mgeerling) [reply src/ui/ServerAdministration/GettingStarted/index.tsx]: I don't know if that will be possible given that we are moving things across domains.

**Pull request merged**